stash-cache/supervisord.d/10-stash-cache.conf

```diff
@@ -4,13 +4,13 @@ user=xrootd
 priority=998
 
 [program:stash-cache]
-command=xrootd -c /etc/xrootd/xrootd-stash-cache.cfg -k fifo -n stash-cache -k 10 -s /var/run/xrootd/xrootd-stash-cache.pid -l /var/log/xrootd/xrootd.log
+command=xrootd -c /etc/xrootd/xrootd-stash-cache.cfg -k fifo -n stash-cache -k 10 -s /var/run/xrootd/xrootd-stash-cache.pid -l /var/log/xrootd/xrootd.log -k %(ENV_XC_NUM_LOGROTATE)s
```

Comment:
So `%{ENV_*}` is how you refer to env variables in supervisor config?

Reply:
Yep see [supervisord configuration](http://supervisord.org/configuration.html)